.storybook/config.js

```diff
@@ -1,5 +1,9 @@
 import { configure, addDecorator } from '@kadira/storybook';
 import { setOptions } from '@kadira/storybook-addon-options';
+import _ from 'underscore';
+
+window._ = _;
```

Comment:
@johnthepink the background image needed it. I was getting an error when the MiniCard tried to set the background image.